crates/astria-sequencer/src/sequencer.rs

```diff
@@ -98,3 +104,44 @@ impl Sequencer {
         Ok(())
     }
 }
+
+fn start_grpc_server(storage: &cnidarium::Storage, grpc_addr: std::net::SocketAddr) -> Result<()> {
+    use ibc_proto::ibc::core::{
+        channel::v1::query_server::QueryServer as ChannelQueryServer,
+        client::v1::query_server::QueryServer as ClientQueryServer,
+        connection::v1::query_server::QueryServer as ConnectionQueryServer,
+    };
+    use penumbra_tower_trace::remote_addr;
+    use tonic::transport::Server;
```

Comment:
It's confusing that this shadows the `Server` import at the module namespace. IMO just `tonic::transport::Server::builder` further down is fine